app/components/referees_review_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<% @application_form.references.each do |referee| %>
+<% @application_form.references.includes(:application_form).each do |referee| %>
```

Comment:
This one surprised me. I thought that the reverse association was populated when you load a collection but doesn't seem to be the case. There is something very odd about `ApplicationForm#references` which has tripped us up before.

`application_form.references.last.application_form` lazy loads `application_form` but `application_form.application_choices.last.application_form` does not. Confused.

Reply:
As discussed its the `-> { order('id ASC') }`

Reply:
Removing the `order` does not stop bullet complaining here. Propose leaving it for now and returning to rename `references` to `application_references` in the hope of reducing the territory this problem has to roam about in?